GVFS/GVFS.Common/Prefetch/Git/GitIndexGenerator.cs

```diff
@@ -109,10 +109,7 @@ private void WriteAllEntries(uint version, HashSet<string> sparseCheckoutEntries
                     LsTreeEntry entry;
                     while (this.entryQueue.TryTake(out entry, Timeout.Infinite))
                     {
-                        bool skipWorkTree = 
-                            sparseCheckoutEntries != null && 
-                            !sparseCheckoutEntries.Contains(entry.Filename) && 
-                            !sparseCheckoutEntries.Contains(this.GetDirectoryNameForGitPath(entry.Filename));
+                        bool skipWorkTree = false;
                         this.WriteEntry(writer, version, entry.Sha, entry.Filename, skipWorkTree, ref lastStringLength);

```

Comment:
It looks like you can remove the `skipWorktree` parameter now? And all of the code inside `WriteEntry` that deals with it.